packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -235,22 +235,6 @@ module.exports = {
             group: BASIC_GROUP,
             description: 'NodeJS flags',
         },
-        {
-            name: 'stats',
-            usage: '--stats <value>',
-            type: [String, Boolean],
```

Comment:
Seems you accidently removed --stats too.

Reply:
😞